fabric-key-binding-api-v1/src/main/java/net/fabricmc/fabric/mixin/client/keybinding/KeyCodeAccessor.java

```diff
@@ -24,6 +24,6 @@
 
 @Mixin(KeyBinding.class)
 public interface KeyCodeAccessor {
-	@Accessor
-	InputUtil.Key getBoundKey();
+	@Accessor("boundKey")
+	InputUtil.Key fabric$getBoundKey();
```

Comment:
We usually use underscore.

Reply:
Changed.

Reply:
> 
> 
> We usually use underscore.

![concern](https://cdn.discordapp.com/emojis/707016186281590855.png?v=1)